utils/validators/validator.js

```diff
@@ -0,0 +1,15 @@
+const joi = require("joi");
+
+const contactSchema = joi.object({
+  name: joi.string().min(3),
+  email: joi.string().email(),
+  phone: joi.string().min(5),
```

Comment:
What is someone types in phone input for example 'Hello'? Is is possible to accept number that comes as a string?